RELEASE_NAME

```diff
@@ -0,0 +1 @@
+Olavtoppen
```

Comment:
This needs to be updated before every release, correct?

Reply:
yes, we could make it a bit more clever by looking in the release message, but it's quite some code and complexity. I don't think it's worth it.

Reply:
Agreed, I thought the same, it was only to be sure :+1: